src/configuration/services/mysql.md

```diff
@@ -1,24 +1,31 @@
 # MariaDB/MySQL (Database service)
 
-MariaDB is a MySQL-compatible relational database system. Its XtraDB storage engine is equivalent to MySQL with InnoDB.
+Platform.sh supports both MariaDB and Oracle MySQL.  While there are some differences at the application level for developers, they function nearly identically from an infrastructure point of view.
 
-See the [MariaDB documentation](https://mariadb.org/learn/) or [MySQL documentation](https://dev.mysql.com/doc/refman/5.5/en/) for more information.
+See the [MariaDB documentation](https://mariadb.org/learn/) or [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/) for more information.
 
 ## Supported versions
 
-* 10.0
-* 10.1
-* 10.2
+The service type `mysql` refers to MariaDB.  The service type `oracle-mysql` refers to MySQL as released by Oracle, Inc.
+
+* mysql:10.0
+* mysql:10.1
+* mysql:10.2
+
+* oracle-mysql:5.7
+* oracle-mysql:8.0
 
 > **note**
 >
-> Downgrades of MariaDB are not supported. MariaDB will update its own datafiles to a new version automatically but cannot downgrade them. If you want to experiment with a later version without committing to it use a non-master environment.
+> Downgrades of MySQL or MariaDB are not supported. Both will update their own datafiles to a new version automatically but cannot downgrade them. If you want to experiment with a later version without committing to it use a non-master environment.
+
+Other than the type, MySQL and MariaDB are otherwise identical and the rest of this page refers to both equally.
```

Comment:
I'm good with this, but I think that this sentence should be above the version lists, something like:

> The service type `mysql` refers to MariaDB.  The service type `oracle-mysql` refers to MySQL as released by Oracle, Inc. Other than the type, MySQL and MariaDB are otherwise identical and the rest of this page refers to both equally.

Otherwise it gets buried to me.